babel.config.json

```diff
@@ -0,0 +1,111 @@
+{
+  // Javascript settings.
```

Comment:
Normally the JSON format doesn't allow comments, which I think is why these comments are highlighted in red

Reply:
> Normally the JSON format doesn't allow comments, which I think is why these comments are highlighted in red

that makes sense. we had to change the extension for workspaces.